posts/2023-02-10-caching-in-github-actions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Speed up workflows with caching in GitHub Actions.
+description: Learn how to use caching in GitHub actions to speed up your workflows.
+author: Han Verstraete
+tags:
+- oss
+- githubactions
+author_img: welteki
+date: "2023-02-10"
+---
+
+Jobs on Actuated runners start in a clean VM each time. This means dependencies need to be downloaded and build artifacts or caches rebuilt each time. GitHub provides a [cache action](https://github.com/actions/cache) that allows caching dependencies and build outputs to improve workflow execution time.
```

Comment:
> Jobs on Actuated runners start in a clean VM each time. This means dependencies need to be downloaded and build artifacts or caches rebuilt each time.

This would be better in the outro than in the intro. The user didn't come here for this, but to learn about caching.

Show them some value

in the conclusion, we get a bit of permission to talk about our solutions.